src/Helper.ts

```diff
@@ -324,6 +343,22 @@ export class Helper {
       manifestGroup.addItem(<IMetadataItem>metadataItem);
     }
 
+    const rights: string | null = this.manifest.getRights();
+    
+    if (rights) {
+      const item: any = {
+        label: "rights",

```

Comment:
Have you checked that this label gets internationalized correctly/appropriately? I'm not entirely sure how i18n works in this code, and there seem to be a few different patterns in play, so I can't tell at a glance if anything is potentially wrong.

Reply:
The label gets replaced with a string from the IIIF data.content strings in iiif-metadata-component (l. 526), following the process that was already in place for 'license', so I'll make sure to check that in iiif-metadata-component/UV.